fedora_messaging/message.py

```diff
@@ -186,6 +186,112 @@ def get_message(routing_key, properties, body):
     return message
 
 
+def dumps(messages):
+    """
+    Serialize messages to a JSON formatted str
+
+    Args:
+        messages (list): The list of messages to serialize. Each message in
+            the messages is subclass of Messge.
+
+    Returns:
+        str: Serialized messages.
+
+    Raises:
+        MsgDumpError: If at least one message is not instance of Message class or subclass, or
+            JSON dump on messages is imposiible.
+    """
+    serialized_messages = []
+    try:
+        for message in messages:
+            message_json = message.dump()
+            serialized_messages.append(message_json)
+    except AttributeError:
+        _log.error("Improper object for messages serialization.")
+        raise MsgDumpError("Message have to be instance of Message class or subclass.")
+
+    try:
+        return json.dumps(serialized_messages)
+    except Exception as e:
+        _log.error("Falied to dump messages: %r", e)
+        raise MsgDumpError(e)
+
+
+def loads(serialized_messages):
+    """
+    Deserialize messages from a JSON formatted str
+
+    Args:
+        serialized_messages (JSON str):
+
+    Returns:
+        list: Deserialized message objects.
+
+    Raises:
+        MsgLoadError: Loading serialized from JSON is imposiible.
+        ValidationError: If deserialized message validation failed.
+    """
+    try:
+        deserialized_messages = [
+            message_str for message_str in json.loads(serialized_messages)
+        ]
+    except json.JSONDecodeError as e:
+        raise MsgLoadError(e)
+
+    messages = []
+    for msg_json in deserialized_messages:
+        try:
+            headers = msg_json["headers"]
+        except KeyError:
+            _log.error("Message saved without headers.")
```

Comment:
I'm inclined to say this should just be an exception - whatever got handed in is not in the format we expect.